dist/index.html

```diff
@@ -6,10 +6,95 @@
+          </div>
+          <div class="container-moto">
+            <h2>Custom Designs · Marketing · Branding · And More </h2>
+          </div>
+        </div>
+      </div>
+      <div class="container-body">
+        <div class="content">
+          <div class="content-left">
+            <span class ="ownerContact" style="color: #336D61; font-size: 3em; font-family: AlexandriaFLF;">
+              CONTACT
+            </span>
+            </h2>
+              <hr>
+            <p style="text-align: center;">
+              <span class="ownerName">
+                LYANDRA SICA<br/>
+              </span>
+              <span class="ownerDetails">
+                Creative Designer<br/><br/>
+                413 Myrtle Avenue <br/>
+                West Islip, NY 11795 <br/>
+                creative@seekadesign.com <br/>
+                631.605.5782 <br/>
+              </span>
+                <hr>
+              <span class="share">
+                FOLLOW. LIKE. SHARE. <br/>
+                <div class="social">
+                  <div class="circle-icon-green">
+                    <a href="https://www.facebook.com/SeekADesign/" target="_blank" class="fab fa-facebook-f"></a> <br/>
+                  </div>
+                  <div class="circle-icon-purple">
+                    <a href="https://www.facebook.com/SeekADesign/" target="_blank" class="fab fa-twitter"></a> <br/>
+                  </div>
+                  <div class="circle-icon-green">
+                    <a href="https://www.instagram.com/seekadesign/" target="_blank" class="fab fa-instagram"></a> <br/>
+                  </div>
+                </div>
+              </span>
+              <span class="at">
+                @SeekADesign
+              </span>
+            </p>
+          </div>
+          <div class="content-right">
+            <h2>Questions?<br/>
+            Shoot Me an Email!</h2>
+            <form action="/my-handling-form-page" method="post">
+              <div>
+                <label for="name"></label>
+                <input type="text" id="name" name="user_name" placeholder="name">
+              </div>
+              <div>
+                <label for="mail"></label>
+                <input type="email" id="mail" name="user_mail" placeholder="email">
+              </div>
+              <div>
+                <label for="name"></label>
+                <input type="text" id="subject" name="user_subject" placeholder="subject">
+              </div>
+              <div>
+                <label for="msg"></label>
+                <textarea id="msg" name="user_message" placeholder="message"></textarea>
+              </div>
+              <div class="container-send">
+                <p class="disclaimer">* All e-mails will be answered in a timely manner. Please include all details when looking for pricing (quality, size, SEND colors, etc.) Marketing e-mails will not be sent to this address.
+                </p>
```

Comment:
Either nest the text between lines 89 and 91, or do this all on one line.

Reply:
I'm not sure what you mean by "nest the text" or put it on one line - do you mean that I should put the < / p > tag on the same line as the opening < p > tag and bring that all in to one line of code?

Reply:
I meant that you can choose from either of the following examples, and either would be acceptable.

**Example of nesting text between tags in HTML:**

```
<p>
  This text is on the next line from its opening tag, and is indented by two spaces.
</p>
```

**Example of putting everything on one line:**

```
<p>This text is coded in-line with respect to the paragraph tags that contain it.</p>

Reply:
I meant that you can choose from either of the following examples, and either would be acceptable.

**Example of nesting text between tags in HTML:**

```
<p>
  This text is on the next line from its opening tag, and is indented by two spaces.
</p>
```

**Example of putting everything on one line:**

```
<p>This text is coded in-line with respect to the paragraph tags that contain it.</p>